kotlin_client/src/test/java/com/github/paylike/kotlin_client/ClientTest.kt

```diff
@@ -101,5 +101,11 @@ class ClientTest {
             "Invalid expiry date, month range is [1..12] and year range is [2000..2099]",
             invalidExpiryException.message
         )
+    }*/
+
+    @Test
+    fun InvalidExpiryExceptionTest() {
```

Comment:
What is this testing?

Reply:
I used it to see what output that exception had. But i delete it because its not needed.